src/utils/object/omit.test.ts

```diff
@@ -0,0 +1,14 @@
+import { omit } from "utils/object";
+
+describe("omit", () => {
+  it("returns a new object without the supplied key", () => {
+    expect(omit<{ b?: number }, ["b"]>({}, ["b"])).toMatchObject({});
```

Comment:
can you remove the typing here? it makes reading it confusing. 

Can you add another test with multiple keys and show it removing only a few of them.